data_steward/validation/participants/validate.py

```diff
@@ -148,6 +77,23 @@ def identify_rdr_ehr_match(client,
     hpo_location_table_id = get_table_id(LOCATION, hpo_id)
     hpo_person_table_id = get_table_id(PERSON, hpo_id)
 
+    street_df = pandas.read_csv(VALIDATION_STREET_CSV, header=0)
+    city_df = pandas.read_csv(VALIDATION_CITY_CSV, header=0)
+    state_df = pandas.read_csv(VALIDATION_STATE_CSV, header=0)
+
+    street_tuples_str: str = ",\n".join([
+        f"('{abbreviated}','{unabbreviated}')" for abbreviated, unabbreviated in
+        zip(street_df['abbreviated'], street_df['unabbreviated'])
+    ])
+
+    city_tuples_str: str = ",\n".join([
+        f"('{abbreviated}','{unabbreviated}')" for abbreviated, unabbreviated in
+        zip(city_df['abbreviated'], city_df['unabbreviated'])
+    ])
+
+    states_str: str = ",\n".join(
+        [f"'{state}'" for state in state_df['abbreviated']])
+
```

Comment:
As discussed, I am not implementing internal tables because BigQuery throws the following error if we use it:
`400 Correlated subqueries that reference other tables are not supported unless they can be de-correlated, such as by transforming them into an efficient JOIN.`
If we want to solve this error, we will need to change how we run the updates quite a bit. We will not make that change for now so that we have enough time to implement both `DC-2130` and `DC-2131` before the next CDR run.